x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/rules/prebuilt_rules/delete_prebuilt_rules_fleet_package.ts

```diff
@@ -7,26 +7,32 @@
 
 import { epmRouteService } from '@kbn/fleet-plugin/common';
 import { PREBUILT_RULES_PACKAGE_NAME } from '@kbn/security-solution-plugin/common/detection_engine/constants';
+import { ToolingLog } from '@kbn/tooling-log';
 import type SuperTest from 'supertest';
 
 /**
  * Delete the security_detection_engine package using fleet API.
  *
  * @param supertest Supertest instance
  */
-export async function deletePrebuiltRulesFleetPackage(supertest: SuperTest.Agent) {
+export async function deletePrebuiltRulesFleetPackage(supertest: SuperTest.Agent, log: ToolingLog) {
```

Comment:
I think functions like `deletePrebuiltRulesFleetPackage` and `deleteEndpointFleetPackage` should implement some retry logic in order to handle flakiness caused by network issues.

Example of a function where we implement retries:

https://github.com/elastic/kibana/blob/4ee52227bce2a1ffffaf930d41513bb0c644c199/x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/rules/prebuilt_rules/delete_all_prebuilt_rule_assets.ts#L12-L29

Reply:
I wasn’t sure if the issue was caused by these API calls, as the error’s stack trace didn’t point to any of our files. Nonetheless, adding retry logic wouldn’t hurt.

Reply:
@xcrzx Addressed in https://github.com/elastic/kibana/pull/205106